node/grpc/server_v2.go

```diff
@@ -180,35 +180,29 @@ func (s *ServerV2) validateAndStoreChunks(
 	ctx context.Context,
 	batch *corev2.Batch,
 	blobShards []*corev2.BlobShard,
-	rawBundles []*node.RawBundles,
+	rawBundles []*node.RawBundle,
 	operatorState *core.OperatorState,
 	batchHeaderHash [32]byte,
 	probe *common.SequenceProbe,
 ) error {
 
-	bundleCount := 0
-	for _, bundles := range rawBundles {
-		bundleCount += len(bundles.Bundles)
-	}
-
-	batchData := make([]*node.BundleToStore, 0, bundleCount)
-	for _, bundles := range rawBundles {
-		blobKey, err := bundles.BlobCertificate.BlobHeader.BlobKey()
+	batchData := make([]*node.BundleToStore, 0, len(rawBundles))
+	for _, bundle := range rawBundles {
+		blobKey, err := bundle.BlobCertificate.BlobHeader.BlobKey()
 		if err != nil {
 			return api.NewErrorInternal("failed to get blob key")
 		}
 
-		for quorum, bundle := range bundles.Bundles {
-			bundleKey, err := node.BundleKey(blobKey, quorum)
-			if err != nil {
-				return api.NewErrorInternal("failed to get bundle key")
-			}
-
-			batchData = append(batchData, &node.BundleToStore{
-				BundleKey:   bundleKey,
-				BundleBytes: bundle,
-			})
+		quorum := core.QuorumID(0)
```

Comment:
Here's another case where I'm not sure what to make of the hardcoded quorum number